### PR TITLE
fix(github-enterprise): Fix github enterprise integration app url 

### PR DESCRIPTION
**Pull request by nima10khodaveisi**
When you try to add configuration for github enterprise app, it redirects you to https://{github-url}/github-apps/ which is not a correct url and returns 404 not found. The correct url is https://{github-url}/apps.

### CODE REVIEW

**Comment by dashed** [comment]
The corrected URL scheme matches docs in here https://docs.github.com/en/enterprise-server@3.4/rest/apps/apps

**Review by dashed — APPROVED** [review]

**Comment by nima10khodaveisi** [comment]
@dashed Can you merge this please?

**Comment by dashed** [comment]
@nima10khodaveisi Before this can be merged, Leander (@leeandher) from @getsentry/ecosystem  will be double checking the GitHub enterprise integration.

**Comment by leeandher** [comment]
Sorry for the delay on this small fix, our test instance of GHE is very out of date and the has some credential issues so we're a bit blocked on testing this.

We're looking into have this updated so we can test/merge this PR and look at https://github.com/getsentry/sentry/issues/34678 as well. Thanks for making this PR, and being patient with this!

**Comment by nima10khodaveisi** [comment]
@leeandher Oh okay thanks for your response. Maybe I could help you with the other issue as well? Let me know if I could help!

**Comment by dashed** [comment]
@leeandher Probably related, but I noticed we do not pass `url` here https://github.com/getsentry/sentry/blob/163fa50513bfd77b34ccf709de0adfdd2c5aba95/src/sentry/identity/github_enterprise/provider.py#L33-L34

since the `get_user_info ()` function consumes it in https://github.com/getsentry/sentry/blob/163fa50513bfd77b34ccf709de0adfdd2c5aba95/src/sentry/identity/github_enterprise/provider.py#L7-L18

**Comment by lifeofgurpreet** [comment]
Looking forward to this fix. Its surprising that we are using Github Teams with the repo belonging to our organization and this feature is not working with Sentry.io. Id imagine most of the business using Sentry are using Github Teams. Understood if it doesn't work for Enterprise but surely it should be working with Github Teams.

**Pull request closed without merging**
